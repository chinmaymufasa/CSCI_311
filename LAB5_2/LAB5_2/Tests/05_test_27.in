2
117
2
194
2
174
2
29
2
153
4
5
6
1
29
-1
