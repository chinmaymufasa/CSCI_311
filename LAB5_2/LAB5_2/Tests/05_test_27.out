174 117 29 153 194 
29 117 153 174 194 
29 153 117 194 174 
29
